opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -40,6 +40,9 @@ pub struct LoggerProvider {
 
 /// Default logger name if empty string is provided.
 const DEFAULT_COMPONENT_NAME: &str = "rust.opentelemetry.io/sdk/logger";
+// According to a Go-specific study mentioned on https://go.dev/blog/slog, 
+// up to 5 attributes is the most common case. We have chosen 8 as the default 
+// capacity for attributes to avoid reallocation in common scenarios.
```

Comment:
no Rust specific study, so picked this! Logging patterns are very similar across languages.